dicedb.conf

```diff
@@ -38,14 +38,6 @@ memory.eviction_ratio = 0.9
 memory.keys_limit = 200000000
 memory.lfu_log_factor = 10
 
-# Persistence Configuration
-persistence.enabled = false
-persistence.aof_file = "./dice-master.aof"
-persistence.persistence_enabled = true
-persistence.write_aof_on_cleanup = false
-persistence.wal-dir = "./"
```

Comment:
We are not using this file. The new config system does not require this at all and is self-sufficient to run on default values and existing files without having to check-in or be present. We can safely delete this file.